### PR TITLE
Add a test suite to create install image for HA

### PR DESCRIPTION
**Pull request by JRivrain**
See https://progress.opensuse.org/issues/116536

gitlab MR https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/400

The new profile is based on https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_sle15/create_hdd/create_hdd_maintenance.xml.ep, but simplified, as the cloned profile triggered a bug.

VR https://openqa.suse.de/tests/9693308#

### CODE REVIEW

File: data/autoyast_sle12/create_hdd/create_hdd_ha.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <!-- Simple install wit HA addon -->
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email>e</email>
```
Comment (jknphy) [inline data/autoyast_sle12/create_hdd/create_hdd_ha.xml:5]: we don't need this field then

File: data/autoyast_sle12/create_hdd/create_hdd_ha.xml [hunk 2]
```diff
@@ -0,0 +1,167 @@
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <!-- Simple install wit HA addon -->
```
Comment (jknphy) [inline data/autoyast_sle12/create_hdd/create_hdd_ha.xml:2]: s/wit/with/

**Review by jknphy — COMMENTED** [review]
we should file that bug for sure.
To review this I would need to compare with a cloned profile from other product to compare and review this.

**Comment by jknphy** [comment]
@chcao @lemon-suse do we have some profile from other product cloned without triggering any bug?

**Comment by lemon-suse** [comment]
> @chcao @lemon-suse do we have some ha ay profile from other product cloned without triggering any bug?

Hi, we haven't such profile since we never install ha via autoyast in previous test.

**Comment by lemon-suse** [comment]
I think the job to create qcow need schedule the schedule/yast/sle_15_sp3/autoyast_create_hdd_maintenance_deregister.yaml to ensure system de-registered.

**Comment by chcao** [comment]
> @chcao @lemon-suse do we have some ha ay profile from other product cloned without triggering any bug?

No, I'm creating a ha profile for 15SPx, it is not ready, still has some issues need to be fixed.

**Comment by jknphy** [comment]
@JRivrain please sync with #15673
Also, please adapt the PR title and commit message to reflect the exact type of migration we are doing here and the product.

**Comment by JRivrain** [comment]
PR replaced by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15750

**Pull request closed without merging**